vrchat.jordo.easyquestswitch/Runtime/Resources/EQS_Localizations/cn.json

```diff
@@ -21,6 +21,7 @@
 	"SettingsHierarchy": "层次结构选项", 
 	"SettingsHierarchyIconShow": "显示图标", 
 	"SettingsHierarchyIconOffset": "隐藏图标",
+	"SettingsPromptForPlatformChange": "Show prompt to apply EQS changes if a newly opened scene still uses setting from the opposite platform (Can cause issues with Bakery).",
```

Comment:
@CallMeReznov If you are available, would you have a translation to suggest for this?

Reply:
I have roughly read your code in "EQS Window. cs"

Can I understand it as,

"If I open a scene in the query platform settings under the Windows platform settings, the EOS switching prompt will pop up."?

Can you upload a picture with this prompt appearing?

Reply:
This is to enable/disable a popup that shows when:
- You setup EQS in one scene
- You switch to another scene
- You change platform (eg from PC to Android)
- You switch back to the original scene where you setup EQS
Because you didn't change platform while you were in the original scene, the original scene is still using PC settings, so a prompt comes up to ask you if you'd like to apply the Android settings like so: 
![image](https://github.com/vrchat-community/EasyQuestSwitch/assets/9134625/ddec22bd-086c-4bae-8f5f-a3d5aa6f01ec)
You've already translated the prompt here: https://github.com/vrchat-community/EasyQuestSwitch/blob/fe1c02ac8c1294b4614ab94ab19fbe79d566febd/vrchat.jordo.easyquestswitch/Runtime/Resources/EQS_Localizations/cn.json#L27

This is to enable/disable that feature in the settings, located right here:
![image](https://github.com/vrchat-community/EasyQuestSwitch/assets/9134625/3534213c-4e30-4e71-8a25-0be3e034ee97)

Reply:
got it,  actually there is no need for such a complex translation

I roughly organize it as

"Is the EQS pop-up application change prompt enabled for scene changes(Remember to manually apply the changes after closing, otherwise it may cause baking problems)"



"SettingsPromptForPlatformChange": "是否打开场景更改时的EQS弹出的应用变更提示 (关闭后需要手动应用变更,否则可能造成烘培错误).",

Reply:
MMmmm, the thing that causes issues with Bakery is the prompt itself. The prompt appearing makes Bakery hang indefinitely when you try to bake until you interact with it, so I'm not sure this translation would be accurate? This is specifically to enable/disable the prompt showing up and giving a slight warning that it may cause Bakery to hang.

Reply:
Are you saying that "pop ups themselves can cause baking abnormalities, that's why you made a switch to manage them"?
if that's the case, then it needs to be modified to

"SettingsPromptForPlatformChange": "是否打开场景更改时EQS弹出的应用变更提示 (启用该功能后在烘培时弹出提示窗口会造成烘培无限挂起,需手动点击弹窗后恢复).",

Reply:
Yes, the popup is useful for some, but for users of the tool Bakery https://assetstore.unity.com/packages/tools/level-design/bakery-gpu-lightmapper-122218 it can cause an issue and hang the process of baking, if you can re-word the issue to be about Bakery specifically I think it'd be perfect 👍

Reply:
"SettingsPromptForPlatformChange": "是否打开场景更改时EQS弹出的应用变更提示 (提示功能与Bakery工具同时启用时,在烘培过程中弹出提示会造成烘培挂起,,需手动点击弹窗后恢复).",

Following your suggestion, I have optimized the translation again

However, I personally believe that there should not be too much text introduction added in this position. On the one hand, I will optimize and abbreviate the text for the introduction of the GUI, and on the other hand, I want to provide a detailed introduction to the origin of the problem

These are two extremes

I suggest adding hyperlinks and creating a page within the plugin's GitHub to carefully explain the issue

For example:

"SettingsPromptForPlatformChange": "是否打开场景更改时EQS弹出的应用变更提示 (与Bakery存在兼容性问题 [详情请点击]https://github.com/vrchat-community/EasyQuestSwitch/pull/32)."

Reply:
Mmm, do you think it's necessary to explain the whole reason on why it's not working? I thought just having "Can cause issues with Bakery" would be enough, and if you have Bakery and get the issue you'll know what this is talking about instantly.

Reply:
just an improvement suggestion, and there are no issues with the current version.

Reply:
Would "是否打开场景更改时EQS弹出的应用变更提示 (与Bakery存在兼容性问题)." be fine then?
Thank you for taking the time to translate this with me, I really appreciate it 😄